### PR TITLE
Users(Create user form): Update create user form with new design

### PR DESCRIPTION
**Pull request by jenny-s51**
## Motivation

https://issues.redhat.com/browse/APPDUX-912

## Verification Steps

1. Go to Users section. 
2. Click "Add user"
3. Create the new user and add groups to the user within the form.
4. Verify that groups selected show up in the "Groups" tab once the user is created.

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [x] axe report has been run and resulting a11y issues have been resolved
- [x] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

### CODE REVIEW

File: src/user/JoinGroupDialog.tsx [hunk 1]
```diff
@@ -95,12 +105,14 @@ export const JoinGroupDialog = ({
   return (
     <Modal
       variant={ModalVariant.small}
-      title={`Join groups for user ${username}`}
+      title={
+        username ? `Join groups for user ${username}` : "Select groups to join"
```
Comment (mfrances17) [inline src/user/JoinGroupDialog.tsx:109]: these strings should be localized

**Review by mfrances17 — CHANGES_REQUESTED** [review]
Functionality is working great... tested creating a new user and adding groups directly in the add. Small change required, you probably want to localize the hardcoded strings.

**Review by edewit — APPROVED** [review]
Apart from the comment of @mfrances17 looking good

File: src/user/JoinGroupDialog.tsx (hunk 1 above)
Comment (jenny-s51) [reply src/user/JoinGroupDialog.tsx]: Nice catch - localized

**Review by mfrances17 — APPROVED** [review]

**Review by mfrances17 — APPROVED** [review]
LGTM, good job

**Pull request merged**